### PR TITLE
Add support for Relaxfm

### PR DESCRIPTION
**Pull request by clasick**
**Describe the changes you made**
Added support for Relaxfm as requested by @pedrojvaz in #2506

### CODE REVIEW

File: src/connectors/relaxfm.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+Connector.playerSelector = '.main-player__wrapp';
+
+Connector.artistSelector = '.TitleExecutorAir';
+
+Connector.trackSelector = '.TitleTracksAir';
+
+Connector.isPlaying = () => $('.toggleplayer').hasClass('stop');
```
Comment (inverse) [inline src/connectors/relaxfm.js:9]: Could be simplified to this.

```suggestion
Connector.pauseButtonSelector = '.toggleplayer.stop';
```

Would avoid the dependency on jQuery which is being phased out in preparation for v3 with bootstrap 5

**Review by inverse — CHANGES_REQUESTED** [review]
Looks good apart from a minor comment :+1:

**Review by alexesprit — APPROVED** [review]

File: src/core/connectors.js [hunk 2]
```diff
@@ -1646,6 +1646,11 @@ const connectors = [{
 	],
 	js: 'connectors/idagio.js',
 	id: 'idagio',
+}, {
+	label: 'Relaxfm',
```
Comment (alexesprit) [inline src/core/connectors.js:1650]: Could you also update the label to "Relax FM", as it's displayed on the website?

Comment (clasick) [reply src/core/connectors.js]: Updated the label.

**Review by inverse — APPROVED** [review]
Nice one :+1:

**Pull request merged**